recipes/openmesh/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "6.2" %}
+
+package:
+  name: openmesh
+  version: {{ version }}
+
+source:
+  fn: OpenMesh-{{ version }}.tar.gz
+  url: http://www.openmesh.org/media/Releases/{{ version }}/OpenMesh-{{ version }}.tar.gz
+  sha256: 570b5b2d3b949050d9628367268c495e87b3dc59f18a07c8037449356fe40374
+  patches:
+    - sitedir.patch  # https://graphics.rwth-aachen.de:9000/OpenMesh/OpenMesh/merge_requests/79
+    - bindings-compilation.patch  # https://graphics.rwth-aachen.de:9000/OpenMesh/OpenMesh/merge_requests/81
+
+build:
+  number: 0
+  features:
+  - vc9  # [win and py27]
+  - vc10  # [win and py34]
+  - vc14  # [win and py35]
+
+# https://graphics.rwth-aachen.de:9000/OpenMesh/OpenMesh/issues/29
+  skip: true  # [win and py35]
```

Comment:
@jakirkham his comment points to an issue that blocks building on 3.5, so it's reasonable to skip it and add support once it get's fixed